src/rgw/rgw_rados.h

```diff
@@ -1090,8 +1090,12 @@ class RGWRados
     ATTRSMOD_MERGE   = 2
```

Comment:
What is going on here?  A PR cannot contain merge commits.  If you want to pull up your whole working set to a newer baseline, you need to cherry-pick them onto a new branch based on the new baseline.  Period.  Upstream will not merge a PR with merge commits.

I am leaving a comment on the first list of this merge commit, because I don't know where else to put it.

Reply:
addressed